init.lua

```diff
@@ -139,25 +138,10 @@ local update_item = function(pos, node)
 		tmp.texture = inv:get_stack("input", 1):get_name()
 		local e = minetest.add_entity(pos, "anvil:item")
 		local yaw = math.pi * 2 - node.param2 * math.pi / 2
-		if e.set_rotation == nil then
-			-- This is for 0.4.16 support, remove it eventually
-			e:set_yaw(yaw)
-			pos.y = pos.y + 5 / 16
-			e:set_pos(pos)
-		else
-			e:set_rotation({x = -1.5708, y = yaw, z = 0}) -- x is pitch, 1.5708 is 90 degrees.
-		end
+		e:set_rotation({x = -1.5708, y = yaw, z = 0}) -- x is pitch, 1.5708 is 90 degrees.
 	end
 end
 
-local metal_sounds
```

Comment:
Shouldn't this be kept?

Reply:
No, the metal sounds were added in 2016 (https://github.com/minetest/minetest_game/commit/fa9a34554b96d2223cce2a9a53991ebe3c5bedc0)